content/advanced/modules/create-example/index.md

```diff
@@ -67,6 +84,176 @@ Create a new YAML file for your module within the `fragments` folder:
 touch fragments/module.yaml
 :::
 
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
```

Comment:
we dont need two different instructions for C9 and local as code is exactly the same, just reword the 

>Open this file in your chosen text editor and paste in the following CloudFormation YAML:

Maybe this could work
> Open this file in the text editor and paste in the following CloudFormation YAML code:

Reply:
Agree... I have updated the and code and sent for review again